utils/include/utils/detail/MonadicOperationWrappers.h

```diff
@@ -87,6 +102,33 @@ detail::or_else_wrapper<T&&> orElse(T&& func) noexcept { return {std::forward<T>
 template<typename T>
 detail::value_or_else_wrapper<T&&> valueOrElse(T&& func) noexcept { return {std::forward<T>(func)}; }
 
+
+/**
+ * Converts from std::optional<T> to nonstd::expected<T, E>
+ * Consuming the expected and discarding the error, if any.
```

Comment:
this second line of the comment is not correct
```suggestion
 * The parameter will be used as the error value if the optional is null.
```